### DISCUSSION TITLE
Is it possible to expose only some api's

### DISCUSSION THREAD

**Discussion by thulasidassrinivasan** [Azure/api-management-developer-portal#1787] [General]
Is it possible to expose only some api's ? 

Say i have 10 API's in my APIM. Is there way i can only expose 5 api's ?
Is it possible to enable ad authentication specific product ? 

Please help :)

**Comment by gpulga** [comment]
Need an answer to this as well

**Comment by gpulga** [comment]
I reached MS support on this and their response was no.

A reference to this can be found in the official documentation: [Tutorial - Access and customize the developer portal - Azure API Management | Microsoft Learn](https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-developer-portal-customize#visibility-and-access-controls).

**Reply by thulasidassrinivasan** [reply]
Thanks @gabrielpulga  for taking time and checking it. I appreciate it.
